src/Microsoft.AspNet.Mvc.Razor/Compilation/RazorCompilationFailure.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.Framework.Runtime;
+
+namespace Microsoft.AspNet.Mvc.Razor
+{
+    /// <summary>
+    /// <see cref="ICompilationFailure"/> for Razor parse failures.
+    /// </summary>
+    public class RazorCompilationFailure : ICompilationFailure
```

Comment:
can't see how this class and `RazorCompilationMessage` changed after you renamed them.  could you please temporarily revert the rename to make other changes visible?  fine to leave the class names as-is during this interim review phase.